modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,124 @@
+= Configure Role-based Access Control
+:description: Role-based access controls provide an extension to ACLs for managing permissions at scale.
+:page-categories: Management, Security
+
+Role-based Access Control (RBAC) provides a way to configure permissions for provisioned users at scale. RBAC allows you to manage access for many users to many resources with a streamlined interface. This mechanism works in conjunction with all supported authentication methods.
+
+IMPORTANT: At present, Redpanda only supports RBAC for self-hosted clusters. Configuration is only supported through the command line using RPK.
+
+== RBAC
+
+RBAC is designed to address the critical challenge of access management at scale. It alleviates the process of manually maintaining and verifying a set of raw permissions (ACLs) for a user base that may contain thousands of users. RBAC provides you with a method to onboard new employees easier, audit accesses faster, and adapt more readily to evolutions of usage and compliance needs.
+
+Using RBAC, you can define roles to reflect organizational structure or job duties to implement a coherent approach to access management. This approach decouples users and permissions through the assignment of roles. This allows a one to many mapping of a given role to many users, thereby dramatically reducing the number of custom policies needed for a resource from one per user to one per group of users.
+
+image::shared:rbac-overview.png[RBAC multiplicity]
```

Comment:
The image kind of just appears--you should refer to it in the narrative. 
eg. The following diagram shows user-to-role mappings using RBAC.
Also--the resolution seems off--the text on the right looks blurry to me. Advise against using internal names like this. Perhaps User1, User 2, and so on. Check with @deniscoady

Reply:
May want to add principal to our glossary too.

Reply:
We're going to need more input from @deniscoady  and @oleiman on this front. I was trying to make the RBAC narrative consistent with the writeups in ACLs which talk about Principals, not Users. In Redpanda, it seems the only type of Principal we have is Users anyway, at least based on what I've seen in the ACL documentation, the PRD, and elsewhere. I defer to Denis on this since we also heard that they want to minimize the use of the term Principal to avoid confusion.